docs/guides/file-system-based-automated-bundle-generation.md

```diff
@@ -180,6 +180,8 @@ The default value of the `auto_load_bundle` parameter can be specified by settin
 
 If server rendering is enabled, the component will be registered for usage both in server and client rendering. In order to have separate definitions for client and server rendering, name the component files as `ComponentName.server.jsx` and `ComponentName.client.jsx`. The `ComponentName.server.jsx` file will be used for server rendering and the `ComponentName.client.jsx` file for client rendering. If you don't want the component rendered on the server, you should only have the `ComponentName.client.jsx` file.
 
+Once generated, all server entrypoints will be imported into a file named `[ReactOnRails.configuration.server_bundle_js_file]-generated.js`, which in turn will be imported into a source file named the same as `ReactOnRails.configuration.server_bundle_js_file`. If your server bundling logic is such that your server bundle source entrypoint is not named the same as your `ReactOnRails.configuration.server_bundle_js_file` & changing it would be difficult, please let us know.
+
```

Comment:
For conditions when server bundle source entrypoint is not named the same as your `ReactOnRails.configuration.server_bundle_js_file`. What should be the solution? Should we document that? I know @frullah faced this issue and had a working solution.

Reply:
There should be no reason that the name needs to be different, AFAIK.

@frullah?

Reply:
@pulkitkkr my issue wasn't related to this, it was different: RoR requires _public_ `server_bundle_js_file` to be present before generating packs. I was able to use `server_bundle_js_file` as a server bundle source entry point.